### PR TITLE
Setup package and pull in schemas from asdf-standard

### PR DESCRIPTION
**Pull request by WilliamJamieson**

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,111 @@
+name: CI
+
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - master
```
Comment (braingram) [inline .github/workflows/ci.yml]: main?

File: CONTRIBUTING.rst [hunk 2]
```diff
@@ -0,0 +1,18 @@
+Reporting Issues
+----------------
+
+Feature requests and bug reports for the Python implementation can be posted at
+`asdf-unit-schemas's github page <https://github.com/asdf-format/asdf-unit-schemas>`_.
```
Comment (braingram) [inline CONTRIBUTING.rst]: Replace with asdf-fits-schemas

File: CONTRIBUTING.rst [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+
+Contributing Code and Documentation
+-----------------------------------
+
+We love contributions! If you're interested in contributing to this project, please open a Pull Request
+or issue and we would be glad to work with you.
+
+.. note::
+    ASDF-unit-schemas is mostly a collection of schemas for the ASDF format. This means that
```
Comment (braingram) [inline CONTRIBUTING.rst]: Replace with asdf-fits-schemas

**Review by braingram — COMMENTED** [review]
Just a few minor comments.

File: .github/workflows/ci.yml (hunk 1 above)
Comment (WilliamJamieson) [reply .github/workflows/ci.yml]: Next time make suggestions via github.

File: docs/manifests.rst [hunk 4]
```diff
@@ -0,0 +1,13 @@
+.. _manifests:
+
+Manifests
+=========
+
+The ASDF tags (described by schemas) available under each ASDF unit schemas version are all described
```
Comment (braingram) [inline docs/manifests.rst]:
```suggestion
The ASDF tags (described by schemas) available under each ASDF fits schemas version are all described
```

File: CONTRIBUTING.rst [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+`asdf-fits-schemas's github page <https://github.com/asdf-format/asdf-fits-schemas>`_.
+
+Contributing Code and Documentation
+-----------------------------------
+
+We love contributions! If you're interested in contributing to this project, please open a Pull Request
+or issue and we would be glad to work with you.
+
+.. note::
+    ASDF-unit-schemas is mostly a collection of schemas for the ASDF format. This means that
```
Comment (braingram) [inline CONTRIBUTING.rst]:
```suggestion
    ASDF-fits-schemas is mostly a collection of schemas for the ASDF format. This means that
```

**Pull request merged**